src/basics.md

```diff
@@ -1385,6 +1386,52 @@ fn main() {
 }
 ```
 
+[ex-examine-date-and-time]: #ex-examine-date-and-time
+<a name="ex-examine-date-and-time"></a>
+## Examine the date and time
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the current UTC [`DateTime`] and its hour/minute/second via [`Timelike`]
+and its year/month/day/weekday via [`Datelike`].
+
+```rust
+extern crate chrono;
+use chrono::{Datelike, Timelike, Utc};
+
+fn main() {
+    let now = Utc::now();
+
+    // available due to chrono::Timelike
```

Comment:
The comment repeats what is said in recipe description (few lines above). Does it make sense to remove it (to make example slightly shorter)?